From f6c7abd45f9e63392ebf8bb299543c90f3642638 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Thu, 17 Sep 2020 10:06:51 +0700
Subject: [PATCH] mk: gcc: allow setting sysroot for libgcc lookup

Makefile fail to find the libgcc file, so that we need append the
directory of libgcc into it.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 mk/gcc.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/gcc.mk b/mk/gcc.mk
index adc77a24..e238b9f3 100644
--- a/mk/gcc.mk
+++ b/mk/gcc.mk
@@ -13,7 +13,7 @@ nostdinc$(sm)	:= -nostdinc -isystem $(shell $(CC$(sm)) \
 			-print-file-name=include 2> /dev/null)
 
 # Get location of libgcc from gcc
-libgcc$(sm)  	:= $(shell $(CC$(sm)) $(CFLAGS$(arch-bits-$(sm))) \
+libgcc$(sm)  	:= $(shell $(CC$(sm)) --sysroot=$(PKG_CONFIG_SYSROOT_DIR) $(CFLAGS$(arch-bits-$(sm))) \
 			-print-libgcc-file-name 2> /dev/null)
 libstdc++$(sm)	:= $(shell $(CXX$(sm)) $(CXXFLAGS$(arch-bits-$(sm))) $(comp-cxxflags$(sm)) \
 			-print-file-name=libstdc++.a 2> /dev/null)
-- 
2.17.1

